packages/inngest/src/components/execution/v1.ts

```diff
@@ -459,44 +460,48 @@ class V1InngestExecution extends InngestExecution implements IInngestExecution {
    * and middleware hooks where appropriate.
    */
   private async startExecution(): Promise<void> {
-    /**
-     * Mutate input as neccessary based on middleware.
-     */
-    await this.transformInput();
+    return getAsyncLocalStorage().then((als) =>
+      als.run({ ctx: this.fnArg }, async (): Promise<void> => {
```

Comment:
When executing, we now wrap the entire operation in our `AsyncLocalStorage.run()` call. This also ensures that middleware is included in the async chain.